docs/manual/getting_started.rst

```diff
@@ -2307,8 +2491,8 @@ be included:
     #include "dds/dds.hpp"
     #include "HelloWorldData.hpp"
 
-At least four DDS entities are needed, the domain participant, the topic
-, the subscriber and the reader.
+At least four DDS entities are needed, the domain participant, 
+the topic, the subscriber, and the reader.
```

Comment:
I think @poetinger mentioned this earlier. We should decide on use of the Oxford comma in a style guide. Many parts of the document do not use it and some of the fixes that were made do not include it.

I personally prefer the Oxford comma though as it limits ambiguity in some cases.